frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -149,7 +149,7 @@ export const navigationLogic = kea<navigationLogicType<WarningType>>({
             },
         ],
     },
-    loaders: {
+    loaders: () => ({
```

Comment:
Q: What does this change? The dayjs invocation is inside another function already, hence it should be lazy or...?

Reply:
🤦 , I think that's right. Removed